setup.py

```diff
@@ -18,6 +18,7 @@
   long_description = fh.read()
 
 install_requires=[
+  'urllib3==1.23', #https://github.com/requests/requests/issues/4830
```

Comment:
Do you want to bump the version number in the same change?

Reply:
for some reason or another it didn't pick up the version change -_-, It should of.

Reply:
I mean the version of django-cloud-deploy